lmnet/lmnet/datasets/bdd100k.py

```diff
@@ -26,7 +28,26 @@ class BDD100K(ObjectDetectionBase):
                "truck"]
     num_classes = len(classes)
     available_subsets = ["train", "validation"]
-    extend_dir = "bdd100k"
+    extend_dir = "BDD100K/bdd100k"
```

Comment:
I guess this change requires extra work to make a new directory `BDD100K`, user would not be able to understand what they should do to make this dataset class works, without reading the source.

Reply:
@tkng San:
I thought about that too. But when you unzip the `BDD100K.zip` data file after downloading from website, it will create a `bdd100k` folder. So, as before may be using only `bdd100k` is better. What do you think?

Reply:
Finally I've used `extended_dir = "bdd100k"` in last commit 643ec047bbd649365df0bcbb42dee57387d4f651

Reply:
Resolved in commit 643ec047bbd649365df0bcbb42dee57387d4f651